script.js

```diff
@@ -151,19 +251,18 @@ function showMoneyQuestion(num) {
 function showTrueAnswers(event) {
   let target = event.target;
   let thisAnswers = document.querySelectorAll('.question-active .question__answers .answer__radio');
-  let falseAnswers = Array.from(thisAnswers).filter(answer => !correctAnswers.includes(answer.value));
-  let randomNum = function() {
-    return Math.floor(Math.random() * 3);
-  }
+  let falseAnswers = Array.from(thisAnswers).filter(answer => {
+     return !correctAnswers.includes(answer.value) && !requestCorrectAnswers.includes(answer.value);
+  });
 
-  let firstRandomNum = randomNum();
+  let firstRandomNum = getRandomNumber(3);
   let secondRandomNum;
   do {
-    secondRandomNum = randomNum();
+    secondRandomNum = getRandomNumber(3);
   } while (firstRandomNum === secondRandomNum)
 
   if (Array.from(target.classList).includes('btn-help__item_opinion-audience')) {
-    falseAnswers[randomNum()].setAttribute('disabled', 'disabled');
+    falseAnswers[getRandomNumber(3)].setAttribute('disabled', 'disabled');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
    falseAnswers[getRandomNumber(3)].setAttribute("disabled", 'disabled');
```